src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -99,6 +99,13 @@
     @Config.RequiresMcRestart
     public static int gasTurbineBonusOutput = 6144;
 
+    @Config.Comment("If true, sounds will be played while machines are active. Default: true")
+    @Config.RequiresWorldRestart
+    public static boolean machineSounds = true;
+
+    @Config.Comment("If true, sounds will be played while using tools. Default: true")
```

Comment:
I would like to suggest a separate and independent config for tool sounds being played in the crafting table. Some sound related mods add sounds that play on craft, and it could be very strange if they were to play simultaneously.